doc/ch-image_desc.rst

```diff
@@ -598,5 +663,18 @@ in the remote registry, so we don't upload it again.)
    cleaning up
    done
 
+:code:`list`
+-----------
+
+List images in storage.
+
+::
+
+  $ ch-image list
+  IMAGE REFERENCE            NAME      TAG        ARCH
+  arm64/v8/debian:stretch    debian    stretch    arm64v8 (aarch64)
+  debian:stretch             debian    stretch    amd64 (x86_64)
+  centos:8                   centos    8          ppcle64
+  centos:7                   centos    7          amd64 (x86_64)
```

Comment:
This is going to be hairy to compute the column widths, and because we don't currently distinguish between name and tag. Possibly:

```suggestion
  $ ch-image list
  arm64/v8/debian:stretch (aarch64)
  debian:stretch (x86_64)
  centos:8 (ppcle64)
  centos:7 (x86_64)
```

Also, why do we have one `debian:stretch` with an architecture path and one without?

Reply:
The image reference is unique. In these examples the image with the arch path was pulled via long-name, e.g., `ch-image pull arm64/v8/debian:stretch` thus exists in storage as `arm64%v8d%ebian:stretch`. 

This also illustrates a key assumption: that architecture guessing is only for short-names, e.g., `debian:stretch`. If you specify a long-name you know what you're doing and what you're asking for. For example, it doesn't make much sense for me to do `ch-image pull ppc64le/centos:8` and get a `x86_64` version.

Reply:
Aha. I don't follow the distinction between short and long names. Seems too magic to me.

Reply:
E.g. what about `foo/bar/debian:stretch`? It's a valid image name but how do we know `foo/bar` isn't an architecture?